src/test/java/ActionsFactoryTest.java

```diff
@@ -0,0 +1,20 @@
+import org.hamcrest.MatcherAssert;
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.*;
+
+public class ActionsFactoryTest {
+
+    @Test
+    public void allMenuOptionsMustHaveAnAction() {
+        MenuOptions[] options = MenuOptions.values();
+
+        for (int i = 0; i < options.length; i++) {
```

Comment:
O que acontece quando chamamos o getAction passando null?  Ou então passando uma opção inválida, que não está em um dos valores do MenuOptions? O ideal é testar um cenário por teste, ou seja, ter um código de teste com loop não é uma boa prática.. tu poderia testar um valor válido, e criar outros testes para os outros cenários, faz sentido?

Reply:
Outro ponto é que tu poderia usar um foreach ao invés de um for nesse caso, o código fica mais legível.